.claude/commands/pr-unresolved.md

```
@@ -0,0 +1,108 @@
+   - For each unresolved thread:
+     - File path
+     - Line number
+     - Full comment body from the thread
+     - Whether it's outdated or collapsed
+
+6. Evaluate the comments and give a justified decision on whether we should fix each one or not.
+7. Ask the user to confirm which ones they want to fix.
+
+## Example Output
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)